conformance/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -108,6 +108,14 @@ object DynamicConformanceJob {
     enabled
   }
 
+  private def isAutocleanStdFolderEnabled()(implicit cmd: CmdConfig): Boolean = {
```

Comment:
This is somewhat unfitting place of the function. Why not to make it a method of `CmdConfig`

Reply:
It checks both Cmd and Config. Currently, Cmd knows nothing about Config.

Reply:
Ah true.